docs/support/latest.rst

```diff
@@ -8,10 +8,13 @@ We might ask you to confirm or update your Moonraker version
 #. Click **HACS**
 #. Click **Integration**
 #. Select **Moonraker** integration
-#. Top corner (left) you should see the version (either ``0.2.0`` or a series of letter and number) |image|
-#. Click 3 little dots top right corner 1. Click **Redownload**
+#. Top corner (left) you should see the version (either ``0.2.0`` or a series of letter and number) (*A*) |image|
+#. Click 3 little dots top right corner
+#. Click **Update Information** (*B*)
+#. Click 3 little dots top right corner
```

Comment:
Repeating this line is intentional.

When you click `Update Info` it closes the menu, so you need to re-open it (thus this step)